scripts/find-props.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+## this script prints out all test/props/prop_*.erl files of a given app,
+## file names are separated by comma for proper's `-m` option
+
+set -euo pipefail
+
+# ensure dir
+cd -P -- "$(dirname -- "$0")/.."
+
+BASEDIR="."
+if [ "$1" != "emqx" ]; then
+    BASEDIR="$1"
+fi
+
+find "${BASEDIR}/test/props" -name "prop_*.erl" -exec basename {} .erl \; 2>/dev/null | xargs | tr ' ' ','
```

Comment:
```suggestion
find "${BASEDIR}/test/props" -name "prop_*.erl" | xargs basename {} .erl \; 2>/dev/null | xargs | tr ' ' ','
```

Reply:
and `| tr -d '\r' | tr '\n' ','` for `xargs | tr ' ' ','`
or change `find-suites.sh` too for consitency

Reply:
I prefer the latter. Because the former would append an extra comma at the end.

Reply:
ok